sdk/python/feast/inference_helpers.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+
+from feast import Entity
+from feast.feature_view import FeatureView
+from feast.value_type import ValueType
+
+
+def infer_entity_value_type_from_feature_views(
+    entities: List[Entity], feature_views: List[FeatureView]
+) -> List[Entity]:
+    incomplete_entities = {
+        entity.name: entity
+        for entity in entities
+        if entity.value_type == ValueType.UNKNOWN
+    }
+    incomplete_entities_keys = incomplete_entities.keys()
+
+    for view in feature_views:
+        if not (incomplete_entities_keys & set(view.entities)):
+            continue
+        col_names_and_types = view.input.get_table_column_names_and_types()
+        for entity_name in view.entities:
+            if entity_name in incomplete_entities:
+                entity_col = list(
```

Comment:
This is called `entity_col` but its a list. Should it be `entity_columns`?

Reply:
yep can rename